### ISSUE TITLE
Enums and traits throw 2 false positives from SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse and SlevomatCodingStandard.Namespaces.UseSpacing.IncorrectLinesCountBeforeFirstUse

### ISSUE THREAD

**Issue by Porthorian** [slevomat/coding-standard#1438]
With Php 8.1 came the inclusion of enums. Some rules don't play nice with enums it seems.

```
[vmarone@vinnielab: ~/projects/fridaycoderelease/aiur/aiur-api]$ php -v
PHP 8.1.11 (cli) (built: Sep 29 2022 16:31:09) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.11, Copyright (c) Zend Technologies
[vmarone@vinnielab: ~/projects/fridaycoderelease/aiur/aiur-api]$ 
```

Occurs in slevomat/coding-standard version 7.2.1

The problematic code.

```
<?php declare(strict_types=1);

namespace FridayCodeRelease\Api\Enum;

enum VerifiedEnum : int implements EnumInterface
{
	use IntTrait;

	case VERIFIED_ON = 1;
	case VERIFIED_OFF = 0;
}
```

The error.
```
[vmarone@vinnielab: ~/projects/fridaycoderelease/aiur/aiur-api]$ make lint
> vendor/bin/phpcs --extensions=php -s '--standard=./../phpcs.xml.dist'

FILE: /home/vmarone/projects/fridaycoderelease/aiur/aiur-api/app/src/Enum/VerifiedEnum.php
----------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------
 7 | ERROR | [x] Type IntTrait is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
 7 | ERROR | [x] Expected 1 line before first use statement, found 0.
   |       |     (SlevomatCodingStandard.Namespaces.UseSpacing.IncorrectLinesCountBeforeFirstUse)
----------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------

Time: 43ms; Memory: 6MB

```

**Comment by Porthorian** [comment]
Duh never mind closing this. Latest version is 8.5 :| so out of date.
